packages/ui/src/stories/molecules/Topbar.stories.tsx

```diff
@@ -33,6 +33,18 @@ export const Default: Story = {
     isLocked: false,
     isLockscreenLoading: false,
     isPasswordSet: true,
+    lock: () => {
+      console.log('Lock action');
+    },
+    toggleDiscreetMode: () => {
+      console.log('Toggling discreet mode');
+    },
+    onNotificationClick: () => {
+      console.log('Notification clicked');
+    },
+    onSyncClick: () => {
+      console.log('Sync clicked');
+    },
```

Comment:
UI does not look good.

![image](https://github.com/Cypherock/cypherock-cysync/assets/80461749/9f75405d-8873-4af6-ad53-eea2e85f73c9)

Reply:
This wasn't done from my end @irshadCypherock

Reply:
Done